samples/web-nodejs-with-db-sample/devfile.yaml

```diff
@@ -17,4 +18,4 @@ commands:
     actions:
       - type: exec
         component: nodejs-app
-        command: cd ${CHE_PROJECTS_ROOT}/nodejs-mongo-app/EmployeeDB/ && npm install && sed -i -- ''s/localhost/mongo/g'' app.js && node app.js
+        command: export HOME=${CHE_PROJECTS_ROOT}; cd ${CHE_PROJECTS_ROOT}/nodejs-mongo-app/EmployeeDB/ && npm install && sed -i -- ''s/localhost/mongo/g'' app.js && node app.js
```

Comment:
If HOME is not writable at all in this container, I would set HOME to projects in this file https://github.com/redhat-developer/devfile/blob/sunix-che-15649/samples/web-nodejs-with-db-sample/nodejs-app.yaml#L34

Reply:
yep good point :)

Reply:
done